legal-api/src/legal_api/version.py

```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '2.135.0'  # pylint: disable=invalid-name
+__version__ = '2.136.0'  # pylint: disable=invalid-name
```

Comment:
This is normally done right before this branch (main) is deployed to Test. Vysakh is handling the deployments this time around. Please check with him on whether to leave this in or not.

Reply:
No need to update. Can revert this change